### DISCUSSION TITLE
Support for Flying Bear Ghost 6

### DISCUSSION THREAD

**Discussion by ultrabybaby** [OrcaSlicer/OrcaSlicer#5725] [Dev]
Good afternoon. It would be nice if you add a pre-configured profile for printing the Flying Bear Ghost 6 printer. Thanks in advance. Because I really want to use your slicer, but I don’t yet have a clear understanding of 3D printing.

**Comment by kybarg** [comment]
see, can download prolfiles for flyingbear and repalce in your folder for now
https://github.com/SoftFever/OrcaSlicer/pull/5826
